### PR TITLE
ACQ-221: Remove 'Stay safe!' from delivery options

### PR DESCRIPTION
**Pull request by remybach**
### Description

[Ticket](https://financialtimes.atlassian.net/browse/ACQ-221)

### CODE REVIEW

File: components/__snapshots__/fieldset.spec.js.snap [hunk 1]
```diff
@@ -26,6 +26,14 @@ exports[`Fieldset renders a custom descriptor paragraph 2`] = `
...
 exports[`Fieldset renders a custom header containing markup 1`] = `
 <fieldset class="ncf__fieldset">
+  <h2 class="ncf__header">
+    <div>
+      Header text
+      <span class="test">
+        Test
+      </span>
+    </div>
+  </h2>
```
Comment (emmalewis) [inline components/__snapshots__/fieldset.spec.js.snap:36]: Is this meant to be here?

File: components/__snapshots__/fieldset.spec.js.snap [hunk 2]
```diff
@@ -35,6 +43,14 @@ exports[`Fieldset renders a custom header containing markup 1`] = `
...
 exports[`Fieldset renders a custom header containing markup 2`] = `
 <fieldset class="ncf__fieldset">
+  <h2 class="ncf__header">
+    <div>
+      Header text
+      <span class="test">
+        Test
+      </span>
+    </div>
+  </h2>
```
Comment (emmalewis) [inline components/__snapshots__/fieldset.spec.js.snap:53]: Is this meant to be here?

File: components/__snapshots__/fieldset.spec.js.snap [hunk 3]
```diff
@@ -157,37 +173,3 @@ exports[`Fieldset renders a fieldset element with default props 2`] = `
   </div>
 </fieldset>
 `;
-
```
Comment (emmalewis) [inline components/__snapshots__/fieldset.spec.js.snap:160]: Intentional?

File: components/__snapshots__/fieldset.spec.js.snap (hunk 1 above)
Comment (remybach) [reply components/__snapshots__/fieldset.spec.js.snap:36]: Good catch... will remove this. Not sure how it snuck in.

**Pull request merged**